icons.yml

```diff
@@ -362,7 +362,7 @@ mappings:
     color: '#EF8000'
   - class: bus
     iconName: bus-11
-    color: '#8665E7'
+    color: '#123295'
```

Comment:
new violet icons need their color to be updated here too:
`#c17900` :arrow_right: `#8665E7`